README.md

```diff
@@ -120,7 +120,18 @@ vars:
     shopify_using_fulfillment_event: true # false by default
 ```
 
-### Step 5: Setting your timezone
+### Step 5: Disable models for non-existent sources
```

Comment:
See my note from the source package about consolidating this with the above section

Reply:
consolidated

Reply:
updated